tools/check_output_test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# Usage: check_output_test.py [path to deno executable]
+import os
+import sys
+import subprocess
+
+root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+tests_path = os.path.join(root_path, "tests")
+
+
+def check_output_test(deno_fn):
+    assert os.path.isfile(deno_fn)
+    outs = [fn for fn in os.listdir(tests_path) if fn.endswith(".out")]
```

Comment:
This implies that "there is a test named "X" if and only if file "X.out" exists.
So every test must have an .out file, or it won't be run.
I think this is not ideal.